AnkiConnect.py

```diff
@@ -502,11 +509,59 @@ def modelFieldNames(self, modelName):
                 return [field['name'] for field in model['flds']]
 
 
-    def multi(self, actions):
-        response = []
-        for item in actions:
-            response.append(AnkiConnect.handler(ac, item))
-        return response
+    def confForDeck(self, deck):
+        if not deck in self.deckNames():
+            return False
+
+        id = self.collection().decks.id(deck)
+        return self.collection().decks.confForDid(id)
+
+
+    def saveConf(self, conf):
```

Comment:
Ah, I see. In that case it makes sense for it to exist. Let's call it `saveDeckConfig`.